packages/camera/src/components/Camera/hooks/useUserMedia.js

```diff
@@ -0,0 +1,43 @@
+import { useState, useEffect } from 'react';
+
+/**
+ * `useUserMedia` is a hook that takes an objects constraints, and returns an object
+ * of `stream` and `error`.
+ */
+function useUserMedia(constraints = { audio: false, video: false }) {
+  const [stream, setStream] = useState(null);
+  const [error, setError] = useState(null);
+
+  useEffect(() => {
+    if (stream) { return; }
+
+    let didCancel = false;
+
+    const getUserMedia = async () => {
+      try {
+        const str = await navigator.mediaDevices.getUserMedia(constraints);
+        if (!didCancel) { setStream(str); }
+      } catch (err) {
+        if (!didCancel) { setError(err); }
+      }
+    };
+
+    getUserMedia();
+    // eslint-disable-next-line consistent-return
```

Comment:
Instead of disabling this eslint error, couldn't we just make the function return `() => {}` on line 12 ?

Reply:
Yes, we can for sure.